src/masks.py

```diff
@@ -13,7 +13,7 @@ def get_mask_card_number(card_number: str) -> str:
 def get_mask_account(account: str) -> str:
     """принимает на вход номер счета и возвращает его маску"""
     if len(account) != 20:
-        print("Вы ввели не верный номер счета")
+        return "Вы ввели не верный номер счета"
```

Comment:
Тут нужно не строку возвращать, а кидать ошибку, например:
```suggestion
       raise ValueError("Вы ввели не верный номер счета")
```

Reply:
что то не работает. По ходу синтаксис raise ValueError до меня не дошел